shell/platform/tizen/public/flutter_tizen.h

```diff
@@ -91,12 +91,10 @@ FlutterDesktopGetPluginRegistrar(FlutterDesktopEngineRef engine,
 FLUTTER_EXPORT FlutterDesktopMessengerRef
 FlutterDesktopEngineGetMessenger(FlutterDesktopEngineRef engine);
 
-/**
- * Return backing window for manipulation in host application.
- *
- * If run engine on a wearable device, need cast void* to Evas_Objct*,
- * otherwise cast void* to Ecore_Wl2_Window*.
- */
+// Returns the window object associated with the Flutter app for manipulation.
```

Comment:
I slightly prefer the original comment because a window is technically associated with the embedder but not the host application, but it's still fine because right now there's only a single window for an app.

Reply:
I'll revert to the original comment so we don't have to change the comment again when multiple windows are supported in the future.